packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Home/Home.tsx

```diff
@@ -82,31 +82,51 @@ const Explore: React.FC<Props> = (props) => {
   const isTablet = useMedia('tablet')
   const contracts = props.openseaApi.includes('testnet')
     ? ['azuki-god', 'dragon-age', 'baychonorarymembers', 'doodles-2sgb43ekw0']
-    : ['with-the-light', 'nouns', 'mfers', 'superrare']
+    : ['nouns', 'mfers', 'superrare']
   const [randomContract] = useState(Math.floor(Math.random() * 4))
   const [assetId, setAssetId] = useState(0)
-  const limit = 4
-  useEffect(() => {
-    const interval = setInterval(() => {
-      if (assetId < 2) {
-        setAssetId(assetId + 1)
-      } else {
-        setAssetId(0)
-      }
-    }, 4000)
-    return () => clearInterval(interval)
-  }, [assetId])
+  const limit = 5
   const [assets] = useAssetsQuery({
     variables: {
       filter: [{ field: AssetFilterFields.CollectionSlug, value: contracts[randomContract] }],
       limit
     }
   })
+  const loadedAssets = assets.data?.assets
+  useEffect(() => {
+    if (loadedAssets && loadedAssets.length > 0) {
+      const interval = setInterval(() => {
+        if (assetId < loadedAssets.length - 1) {
+          setAssetId(assetId + 1)
+        } else {
+          setAssetId(0)
+        }
+      }, 4000)
+      return () => clearInterval(interval)
+    }
+  }, [assetId, loadedAssets])
   const [results] = useCollectionsQuery({
     variables: {
       sort: { by: CollectionSortFields.OneDayVolume, direction: SortDirection.Desc }
     }
   })
+  const Carousel = loadedAssets
+    ? loadedAssets.map((asset, i) => {
+        if (!asset.image_url) return null
+        return (
+          <Text
+            // eslint-disable-next-line react/no-array-index-key
+            key={i}
```

Comment:
maybe to change this to use `asset.image_url` instead of `i` or something from asset object?